### PR TITLE
Cypress test cases for vertical and pie charts

### PR DESCRIPTION
**Pull request by Koustubh5585**
### Description
Cypress test cases for vertical and pie charts with 2-way data sync

### Issues Resolved
https://github.com/opensearch-project/observability/issues/1055
https://github.com/opensearch-project/observability/issues/1056

### Check List
- [ ] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: dashboards-observability/public/components/event_analytics/explorer/visualizations/config_panel/config_panel.scss [hunk 1]
```diff
@@ -164,4 +164,3 @@ $vis-editor-sidebar-min-width: 350px;
 #vis__mainContent .vis__leftPanel {
   overflow-y: unset; // unset default setting
 }
-
```
Comment (ruchika-narang) [inline dashboards-observability/public/components/event_analytics/explorer/visualizations/config_panel/config_panel.scss:167]: Add extra line at end of file.

File: dashboards-observability/.cypress/integration/VisualizationCharts/1_verticalBar_chart.spec.js [hunk 2]
```diff
@@ -0,0 +1,246 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/// <reference types="cypress" />
```
Comment (ruchika-narang) [inline dashboards-observability/.cypress/integration/VisualizationCharts/1_verticalBar_chart.spec.js:6]: Rename this file with using proper underscores. Eg: 1_vertical_bar_chart.spec.js

**Review by ruchika-narang — CHANGES_REQUESTED** [review]
Please rename the branch.

**Comment by Koustubh5585** [comment]
> Please rename the branch.

What should be the name

File: dashboards-observability/public/components/event_analytics/explorer/visualizations/config_panel/config_panel.scss (hunk 1 above)
Comment (Koustubh5585) [reply dashboards-observability/public/components/event_analytics/explorer/visualizations/config_panel/config_panel.scss]: resolved

File: dashboards-observability/.cypress/integration/VisualizationCharts/1_verticalBar_chart.spec.js (hunk 2 above)
Comment (Koustubh5585) [reply dashboards-observability/.cypress/integration/VisualizationCharts/1_verticalBar_chart.spec.js:6]: Renamed

**Pull request closed without merging**